docs/javascript-installation.md

```diff
@@ -0,0 +1,114 @@
+---
+id: javascript-installation
+title: Scope Javascript Agent installation
+sidebar_label: Installation
+---
+
+Installation of the Scope Agent is done via [npm](https://www.npmjs.com/package/@undefinedlabs/scope-agent).
+
+```bash
```

Comment:
We can use [docusaurus code tabs feature](https://docusaurus.io/docs/en/doc-markdown#language-specific-code-tabs) here